fileuploads/templates/fileuploads/about.html

```diff
@@ -3,18 +3,39 @@
 
 <span>
     <div class = "container-fluid">
-    <h1>About signalserver</h1>
+    <h1>About SignalServer</h1>
     <div class="row">
-        <div class="col-md-6">
-          <p>
-            signalserver is a web application that significantly reduces the amount of time required to perform quality control on large sets of video files. Built upon QCTools and FFmpeg, signalserver allows users to set their own Pass/Fail parameters, cross-referencing QCTools reports in an effort to identify potentially problematic outliers. Like MediaConch, but focused on frame-based video analysis rather than technical metadata policy checking, signalserver reduces human error by providing comprehensive and customizable collection-level quality control reporting.
-          </p>
-          <p>
-            signalserver is free and open source, and would not exist without the National Endowment for the Humanities, the Knight Foundation, and the FFmpeg community.
-          </p>
+        <div class="col-md-6"><p>
+             <a href="https://github.com/bavc/signalserver">SignalServer</a> is a web application that significantly reduces the amount of time required to perform quality control on large sets of video files. Built upon <a href="https://bavc.org/preserve-media/preservation-tools">QCTools</a> and <a href="https://ffmpeg.org/">FFmpeg</a>, the application allows users to set their own pass/fail parameters, cross-referencing QCTools reports in an effort to identify potentially problematic outliers. Like <a href="https://mediaarea.net/MediaConch/">MediaConch</a>, but focused on frame-based video analysis rather than technical metadata policy checking, SignalServer reduces human error by providing comprehensive and customizable collection-level quality control reporting.
+             <p>
```

Comment:
<p> tags be at the beginning of a paragraph and should close at the end of the paragraph with </p>